pylearn2/scripts/datasets/make_svhn_pytables.py

```diff
@@ -0,0 +1,36 @@
+"""
+Script to split the downloaded SVHN .mat files to 'test', 'valid', and 'train'
+sets and save them as pytables. Ensure that you have set the PYLEARN2_DATA_PATH
+environment variable and downloaded the .mat files to the path
+${PYLEARN2_DATA_PATH}/SVHN/format2/
+"""
+import os
+from pylearn2.datasets.svhn import SVHN
+from pylearn2.utils.string_utils import preprocess
+
+assert 'PYLEARN2_DATA_PATH' in os.environ, "PYLEARN2_DATA_PATH not defined"
```

Comment:
Could you wrap everything but "import" statements into `if __name__ == '__main__'`?